### PR TITLE
ADDED get_mixing_state_index

### LINKED ISSUE #723: Add Mixing state index get_func
Add mixing state index calculation function under particle properties

### PR DESCRIPTION
**Pull request by Gorkowski**
- Ported over the get_mixing_state_index calculation from the paper example code.
- added test
- exposed in int

Fixes #723 

## Summary by Sourcery

Add a new function to calculate the aerosol mixing state index (χ) for particle populations

New Features:
- Implement get_mixing_state_index function to quantify the mixing state of aerosol particles based on their species masses

Tests:
- Add comprehensive test cases for mixing state index calculation, covering internal mixture, external mixture, single particle, and edge cases

## Summary by Sourcery

Add a new function to calculate the aerosol mixing state index (χ) for particle populations

New Features:
- Implement get_mixing_state_index function to quantify the mixing state of aerosol particles based on their species masses, providing a metric to assess how internally or externally mixed an aerosol population is

Enhancements:
- Increment package version to 0.2.5 to reflect the new functionality

Tests:
- Add comprehensive test cases for mixing state index calculation, covering internal mixture, external mixture, single particle, and edge cases

### CODE REVIEW

File: particula/particles/properties/mixing_state_index.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+from numpy.typing import NDArray
+import numpy as np
+
+from particula.util.machine_limit import get_safe_log10, get_safe_exp
+from particula.util.validate_inputs import validate_inputs
+
+
+@validate_inputs({"species_masses": "nonnegative"})
+def get_mixing_state_index(
```
[the remark this answers is not in the document]
Comment (Gorkowski) [reply particula/particles/properties/mixing_state_index.py:14]: doesn't seem to happen in all identical test

File: particula/particles/properties/tests/mixing_state_index_test.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    chi = get_mixing_state_index(masses)
+    assert np.isnan(chi)
+
+
+def test_negative_mass_raises_value_error():
+    """
+    Negative inputs are invalid and should raise.
+    """
+    masses = np.array([[1.0, -0.1]])
+    with pytest.raises(ValueError):
```
[the remark this answers is not in the document]
Comment (Gorkowski) [reply particula/particles/properties/tests/mixing_state_index_test.py:66]: to much testing

**Review by wkchuang — APPROVED** [review]
Nice addition
